src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/handlers/PrisonerReleasedEventHandler.kt

```diff
@@ -29,7 +29,12 @@ class PrisonerReleasedEventHandler(
   }
 
   private fun cancelFutureBookingsFor(prisonerNumber: String) {
+//    prisonAppointmentRepository.findByPrisonerNumber(prisonerNumber)
     prisonAppointmentRepository.findPrisonerPrisonAppointmentsAfter(prisonerNumber, LocalDate.now(), LocalTime.now())
+      .ifEmpty {
+        log.info("No future bookings currently exists for prisoner $prisonerNumber")
```

Comment:
Could probably remove this?

Reply:
Might come in handy? Maybe leave for now, and if ceases to be useful remove later?